files/InfinumCodeStyle.xml

```diff
@@ -65,15 +66,13 @@
         <package name="" alias="true" withSubpackages="true" />
       </value>
     </option>
-    <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
-    <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="2147483647" />
```

Comment:
I guess this was redundant then 🤔 

Could we also remove ktor star imports? What was the point in allowing them?
https://github.com/infinum/android-handbook/pull/111/files#diff-9b1d220a420ec9713b33785a3153f8056a84f776ddac50ae65590aa355a747b4L59

Reply:
I have never used Ktor so I would leave that decision up to someone that has.

Reply:
We use it. I turned that off manually because it was in conflict with detekt rules.
I suggest we remove it from here as well.